### PR TITLE
Added routes to appbar with navigation functionality

### PR DESCRIPTION
**Pull request by devin-mcewan**
Card: (https://github.com/orgs/HTMElephant/projects/2/views/1?pane=issue&itemId=47754595)

Routes currently navigate to the desired route, but the endpoint does not exist yet. 
Added additional ":hover" styling so we know we are on the button.

### CODE REVIEW

File: src/components/ResponsiveAppBar/ResponsiveAppBar.js [hunk 1]
```diff
@@ -4,22 +4,27 @@ import AppBar from "@mui/material/AppBar";
...
 import { useNavigate } from "react-router";
+import "./ResponsiveAppBar.css";
 
 const ResponsiveAppBar = () => {
   const navigate = useNavigate();
   const handleClick = () => {
     navigate("/login");
   };
+  const handleNavigate = (route) => {
+    navigate(`${route}`);
```
Comment (jacobdman) [inline src/components/ResponsiveAppBar/ResponsiveAppBar.js:15]: I like what you're doing - but it's over engineered! `route` is already a string - no need to wrap it in a string!

```suggestion
    navigate(route);
```

File: src/components/ResponsiveAppBar/ResponsiveAppBar.js [hunk 2]
```diff
@@ -37,6 +42,52 @@ const ResponsiveAppBar = () => {
...
+                component="a"
+                sx={{
+                  mr: 2,
+                  display: { xs: "none", md: "flex" },
+                  fontFamily: "monospace",
+                  fontWeight: 700,
+                  letterSpacing: ".3rem",
+                  color: "inherit",
+                  textDecoration: "none",
+                }}
```
Comment (jacobdman) [inline src/components/ResponsiveAppBar/ResponsiveAppBar.js:62]: this `sx` is repeated - is there a better way to do styling so it's 
1 - not inline
2 - re-useable? 🤔 (you're doing it with this typography already!!!)

File: src/components/ResponsiveAppBar/ResponsiveAppBar.js (hunk 1 above)
Comment (devin-mcewan) [reply src/components/ResponsiveAppBar/ResponsiveAppBar.js:22]: fixed!

File: src/components/ResponsiveAppBar/ResponsiveAppBar.js (hunk 2 above)
Comment (devin-mcewan) [reply src/components/ResponsiveAppBar/ResponsiveAppBar.js]: Added to CSS!

**Review by jacobdman — APPROVED** [review]
Great work - super clean!

**Review by Naphrum — APPROVED** [review]
This looks great!

**Pull request merged**